.ci_install_p4est.sh

```diff
@@ -4,35 +4,16 @@ if [ -z "$P4EST_RELEASE" ]; then
   P4EST_RELEASE="2.8"
 fi
 
-if [ "${P4EST_TEST}" = "P4EST_JLL_USES_MPI_PRE_GENERATED_BINDINGS" ]; then
-  echo "Found 'P4EST_TEST=${P4EST_TEST}'. Nothing to do here."
+if [ "${JULIA_P4EST_TEST}" = "P4EST_JLL_MPI_DEFAULT" ]; then
+  echo "Found 'JULIA_P4EST_TEST=${JULIA_P4EST_TEST}'. Nothing to do here."
 fi
-if [ "${P4EST_TEST}" = "P4EST_JLL_USES_MPI" ]; then
-  echo "Found 'P4EST_TEST=${P4EST_TEST}'. Nothing to do here."
+if [ "${JULIA_P4EST_TEST}" = "P4EST_JLL_MPI_OPENMPIJLL" ]; then
+  echo "Found 'JULIA_P4EST_TEST=${JULIA_P4EST_TEST}'. Nothing to do here."
```

Comment:
The variable P4EST_JLL_MPI_OPENMPIJLL is only ever mentioned here. Should this be removed then? Or is this a test that we actually should run but is currently disabled?
```suggestion
```

Reply:
Initially, I thought this might be a nice test. However, we would need to build P4est_jll.jl with OpenMPI_jll.jl support, which you did not want to do.
Removed in 31d1477